sources/web/datalab/polymer/test/file-browser-test.ts

```diff
@@ -44,6 +44,20 @@ describe('<file-browser>', () => {
     new MockFile('file3'),
   ];
 
+  const fileAddButtonIds = [
```

Comment:
If you call this 'alwaysEnabledButtonIds', you could put uploadButton in this list, which would allow you to simplify some of the testing code.

Reply:
Done.